app/src/attendance/attendance_check.py

```diff
@@ -0,0 +1,74 @@
+from timezonefinder import TimezoneFinder
+from pydantic import constr
+
+from app.models import Coordinate, AttendanceCheckResponse
+from app.src.database.fake_database import fake_db
+from app.src.types import attendance_check_in_type
+from geopy.geocoders import Nominatim
+from datetime import datetime
+import pytz
+
+
+class AttendanceCheck:
+    """AttendanceCheck: 출퇴근 신청 클래스"""
+    DEFAULT_ACCOUNT_ID: constr = "freeworksquad"
+    DEFAULT_COORDINATE: Coordinate = Coordinate(latitude=0.0, longitude=0.0)
+
+    def __init__(
+            self,
+            account_id: constr,
+            coordinate: Coordinate
+    ):
+        self.coordinate = coordinate if coordinate is not None else AttendanceCheck.DEFAULT_COORDINATE
+        self.account_id = account_id if account_id is not None else AttendanceCheck.DEFAULT_ACCOUNT_ID
+
+    def attendance_check_in(self) -> attendance_check_in_type:
```

Comment:
이름 명명 규칙에서 동사형태가 시작되어야 할 것 같은데 
check_in_~~ 형태는 어떨까요?

Reply:
좋습니다! 👍check_in_attendance가 더 직관적일 꺼 같습니다! 

```suggestion
    def check_in_attendance(self) -> check_in_attendance_type:
```